### ISSUE TITLE
Request for Pre-trained Models / Checkpoints

### ISSUE THREAD

**Issue by lusheng211** [xiaomiaozhang/Co-MTP#4]
When I'm training the model myself, the loss keeps throwing various errors, and I don't know why. Could you please provide the original model and weights? Thank you.
